src/languages/en.ts

```diff
@@ -846,7 +840,10 @@ const translations = {
         receiptScanning: 'Receipt scanning...',
         receiptScanInProgress: 'Receipt scan in progress',
         receiptScanInProgressDescription: 'Receipt scan in progress. Check back later or enter the details now.',
-        receiptIssuesFound: ({count}: DistanceRateOperationsParams) => `${count === 1 ? 'Issue' : 'Issues'} found`,
+        receiptIssuesFound: () => ({
+            one: `Issue found`,
```

Comment:
Yeah:
> Going to approve since I posted this to see if we enable the eslint rule for it: https://expensify.slack.com/archives/C01GTK53T8Q/p1727441233067399